lib/ansible/vars/manager.py

```diff
@@ -141,7 +141,8 @@ def extra_vars(self):
     def set_inventory(self, inventory):
         self._inventory = inventory
 
-    def get_vars(self, play=None, host=None, task=None, include_hostvars=True, include_delegate_to=True, use_cache=True):
+    def get_vars(self, play=None, host=None, task=None, include_hostvars=True, include_delegate_to=True, use_cache=True,
```

Comment:
There was also some discussion about whether we could somehow hang the cache off the play object instead of adding the new optional args everywhere. Left as an exercise for the reader, but the new plumbing for this is the "ickiest" part of this change IMO.

Reply:
Yeah, I don't really disagree.  I'm not necessarily a huge fan of these extra args either, but it didn't feel right hanging it off the play either.